Automattic/wp-calypso
# Add user Akismet API key on the purchase page

## Proposed Changes

* This PR adds the user's Akismet API key to Akismet purchases in the purchase management section

## Testing Instructions

If D105598-code has not been merged yet, you will need to test with that diff applied.
 
* Apply this PR and spin up your local Calypso testing environment
* You will need a test Akismet purchase to test this PR. If you do not have a test purchase already, you can go to `calypso.localhost:3000/checkout/akismet/ak_plus_yearly_1` to buy a test product (do this in test mode).
* Once you have a sample purchase, navigate to `/me/purchases` and select the Akismet purchase you just acquired.
* On the Purchase management screen, you should see your Akismet API key showing just below the purchase data. You should be able to use the "Copy" button to copy the key to the clipboard.

Regression testing
* Test with Jetpack license purchases to verify that the license keys still display as expected on the purchase pages. This PR modifies the component that was being used to display the license keys.

## Pre-merge Checklist
